quantumdb-postgresql/src/main/java/io/quantumdb/core/planner/SelectiveMigratorFunction.java

```diff
@@ -54,7 +54,7 @@ private static MigratorFunction createUpdateMigrator(RefLog refLog, Table source
 				.collect(Collectors.toList());
 
 		List<String> functionParameters = primaryKeyColumns.stream()
-				.map(column -> functionParameterMapping.get(quoted(column.getName())) + " " + column.getType().toString())
```

Comment:
Why is this wrong? Now the resulting query (when given a column with uppercase characters) will be: `CREATE FUNCTION "migrator_1234567"(ID bigint) ...` for instance instead of `CREATE FUNCTION "migrator_1234567"("ID" bigint) ...`

Reply:
That is because you request the value (q1,q2,q3) with the column name as the key. See below code how functionParameterMapping is made
```
		List<Column> primaryKeyColumns = source.getPrimaryKeyColumns();
		Map<String, String> functionParameterMapping = Maps.newHashMap();
		for (int i = 0; i < primaryKeyColumns.size(); i++) {
			functionParameterMapping.put(primaryKeyColumns.get(i).getName(), "q" + i);
		}
```
The function is created with q1, q2, q3 as the parameters, e.g. FUNCTION "migrator_1234567"(q1 bigint, q2 integer). Not the column names.

In this file there are 2 functions (createInsertMigrator and createUpdateMigrator) and one had the quoted method and the other not, the tests failed on this function because it could not get the q1, q2 as the keys were first made without quotes and later retrieved with quotes.